secure/key-manager/src/tests.rs

```diff
@@ -208,6 +203,11 @@ impl LibraInterface for TestLibraInterface {
             .microseconds)
     }
 
+    fn last_reconfiguration(&self) -> Result<u64, Error> {
```

Comment:
why? they were in alphabetical order.

Reply:
Oh, well CLion pointed out that the implementation order doesn't match the order defined by the LibraInterface. It's a tiny point but I wanted to get rid of the squiggly lines in my IDE ;)

Reply:
I think the better approach would have been to fix the ordering in the trait.